LIC_FILES_CHKSUM.sha256

```diff
@@ -15,6 +15,7 @@ a284e538f3e5649ae6a8f32da2bbd18e9daabf2c1ebbfb99ed92e797b159be1a  vendor/github.
 # ISC licenses.
 1b93a317849ee09d3d7e4f1d20c2b78ddb230b4becb12d7c224c927b9d470251  vendor/github.com/davecgh/go-spew/LICENSE
 a94710b55e03b5285f77d048c5ba61bb9d6ee04a06c0eb90e68821e11b0c707a  vendor/gopkg.in/yaml.v2/LICENSE.libyaml
+16f848582e4b276a7392cd34496b7a33d6f65c0e190c163ff3a056a7c61219ce  vendor/github.com/klauspost/compress/LICENSE
```

Comment:
Please add this under BSD 3 Clause licenses.

Reply:
How did you know it was a BSD 3 license?

Reply:
Years of experience, my friend!

No, seriously, they're always quoting the same license, so it's pretty easy to recognize it once you've looked at a few. The "3 Clause" part is simply the three bullet points, and I skimmed to see that the text was correct.

Reply:
It would be more impressive if you recognized it by the SHA256 :)

Reply:
hahaha, that would have been an amazing bluff. I might have fallen for that one actually :p